connect/KymaConnector.go

```diff
@@ -0,0 +1,84 @@
+package connect
+
+import (
+	"encoding/json"
+	"github.com/kyma-incubator/hydroform/connect/types"
+	"net/http"
+)
+
+type KymaConnector struct {
+	CsrInfo          *types.CSRInfo
+	Ca               *types.ClientCertificate
+	Info             *types.Info
+	SecureClient     *http.Client
+	StorageInterface WriterInterface
+}
+
```

Comment:
I think we should make the writer interface more explicit. More in format of getter and setter (e.g. getKey, setKey or getCrt and setCrt. By doing so, usecases like databases as storage provider will be easier to implement.

Reply:
Hi, I have updated the interface to be more specific to the functions now.